test/plugins/features/movies/controller.test.js

```diff
@@ -1,10 +1,30 @@
 'use strict';
 
-const Controller = require('../../../../lib/plugins/features/movies/controller');
-const Movie      = require('../../../../lib/models/movie');
+const Bluebird = require('bluebird');
+
+const Knex = require('../../../../lib/libraries/knex');
+
+const Controller   = require('../../../../lib/plugins/features/movies/controller');
+const Movie        = require('../../../../lib/models/movie');
+const MovieFactory = require('../../../factories/movie');
+
+const movie1 = MovieFactory.build({ name: 'Star Trek', release_year: 1982 });
+const movie2 = MovieFactory.build({ name: 'Star Trek 2', release_year: 1984 });
+const movie3 = MovieFactory.build({ name: 'Star Trok', release_year: 1986 });
 
 describe('movie controller', () => {
 
+  beforeEach(() => {
+    return Bluebird.all([
+      Knex.raw('TRUNCATE movies CASCADE')
```

Comment:
Since there is only one promise in the array of promises, it doesn't need to be wrapped in a `Bluebird.all`:

```js
return Knex.raw('TRUNCATE movies CASCADE')
.then(() => {
  // ...
```